packages/hardhat/contracts/ToKanBan.sol

```diff
@@ -32,30 +46,29 @@ contract ToKanBan{
         bool assigned;
         address payable raider;
         bool reviewed;
+        mapping(address => bool) approvals;
         bool closed;
     }
 
     // Modfier restricting access to PM
     modifier onlyPM(){
-        require(pm == msg.sender,"Only the ");
+        require(pm == msg.sender,"Only the PM can access this ");
```

Comment:
This will probably clash with my earlier changes that are now on master, you might want to remove this change from the PR. (@ me if you aren't sure how to do that and I can show you)

Reply:
Deleted my change